lib/screens/profile.dart

```diff
@@ -0,0 +1,272 @@
+                    ),
+                    SizedBox(
+                      height: 10,
+                    ),
+                    Text(
+                      widget.email,
+                      style: TextStyle(
+                          fontSize: 2.1 * SizeConfig.textMultiplier,
+                          color: ThemeColoursSeva().pallete1),
+                    )
+                  ],
+                )),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.all(10.0),
+              child: Card(
+                child: Column(
+                  mainAxisSize: MainAxisSize.min,
+                  children: <Widget>[
+                    const ListTile(
+                      title: Text('My Orders'),
+                    ),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.end,
+                      children: <Widget>[
+                        const SizedBox(width: 8),
+                        TextButton(
+                          child: const Text('VIEW ALL ORDERS'),
+                          onPressed: () {
+                            Navigator.of(context).push(CupertinoPageRoute<Null>(
+                                builder: (BuildContext context) {
+                              return NewOrdersScreen();
+                            }));
+                          },
+                        ),
+                        const SizedBox(width: 8),
+                      ],
+                    ),
+                  ],
+                ),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.all(10.0),
+              child: Card(
+                child: Column(
+                  mainAxisSize: MainAxisSize.min,
+                  children: <Widget>[
+                    const ListTile(
+                      title: Text('Location'),
+                    ),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.end,
+                      children: <Widget>[
+                        const SizedBox(width: 8),
+                        TextButton(
+                          child: const Text('VIEW MY LOCATION'),
+                          onPressed: () {
+                            _showLocation(context);
+                          },
+                        ),
+                        const SizedBox(width: 8),
+                      ],
+                    ),
+                  ],
+                ),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.all(10.0),
+              child: Card(
+                child: Column(
+                  mainAxisSize: MainAxisSize.min,
+                  children: <Widget>[
+                    ListTile(
+                      title: Center(
+                        child: TextButton(
+                          child: const Text('Logout'),
+                          onPressed: () async {
```

Comment:
If **logout** button is here, then why is it in `sidenav` as well ?

Reply:
I will remove from sidenav

Reply:
or we can give at both places.